tests/integration/test_int_dataframe.py

```diff
@@ -2672,3 +2672,19 @@ def test_alias_group_by_column(
     dfs = dfs.groupBy(SF.col("name").alias("Firstname")).count()
 
     compare_frames(df, dfs, compare_schema=False, sort=True)
+
+
+def test_is_empty(
```

Comment:
Probably best to move this function too to that other file (see #362 for details)